addons/l10n_ch/models/res_bank.py

```diff
@@ -184,6 +186,9 @@ def _is_qr_iban(self):
         QR-codes. They are formed like regular IBANs, but are actually something
         different.
         """
+        if not self:
+            return False
```

Comment:
Is there a legit case where we try to call it on an empty record ?

Reply:
This is to simplify the call to `_is_qr_iban` by replacing:

`self.invoice_partner_bank_id and self.invoice_partner_bank_id._is_qr_iban()`

by

`self.invoice_partner_bank_id._is_qr_iban()`

In master we might want to consider detecting a QR-IBAN by an other mean. Either with a computed field or as a separate type from IBAN.